tests/libvmi/factory.go

```diff
@@ -73,12 +76,20 @@ func NewAlpine(opts ...Option) *kvirtv1.VirtualMachineInstance {
 }
 
 func NewAlpineWithTestTooling(opts ...Option) *kvirtv1.VirtualMachineInstance {
+	alpineMemory := cirrosMemory
```

Comment:
What about having an explicit function for `alpineMemory` please ?
both can call a common helper which does the function body (line 91)
or can simply call the `cirrosMemory`

Reply:
What is the problem with an alias?

Reply:
I find it less intuitive / less nice,
preferable imo a function that calls the other function or such

Reply:
It is in the boundary of a few lines, I think it is readable.
- Common function: I do not have a good name for it.
- Wrapping with a function: I do not think it is worth the stack & code overhead.

Reply:
> * Common function: I do not have a good name for it.

`vmiMemory()` or something likewise ?

Reply:
> `vmiMemory()` or something likewise ?

It will be incorrect as we have Fedora which does not confirm to this at all.

> It will be better to have this helper / common because it is not more accurate than having
> cirrosMemory and aliasing it to alpine imo (semantically cirros is the subject where the subject is common)

Sorry, but this is not convincing me.
A common function I cannot describe well and a wrapper has no advantage over the simple aliasing.
Said that, if the maintainer that will approve this change will require it, I will comply for the sake of getting it in.

Reply:
> It will be incorrect as we have Fedora which does not confirm to this at all.

Fwiw, both Cirros and Alpine are lightweight Linux so having this in the name can make it more accurate if desired

Reply:
> Fwiw, both Cirros and Alpine are lightweight Linux so having this in the name can make it more accurate if desired
> (Cirros is consider minimal Linux as well)

I do not understand how it is related to the mentioning of Fedora, sorry.

Reply:
> > Fwiw, both Cirros and Alpine are lightweight Linux so having this in the name can make it more accurate if desired
> > (Cirros is consider minimal Linux as well)
> 
> I do not understand how it is related to the mentioning of Fedora, sorry.

They are lightweight Linux, whereas Fedora isn't